packages/blade/src/components/Table/Table.web.tsx

```diff
@@ -408,6 +484,14 @@ const _Table = <Item,>({
     }
   }
 
+  if (selectionType !== 'none' && hasHoverActions && __DEV__) {
+    // their is no point of using hover actions with selectionType
```

Comment:
oh will it cause any issues in selectionType?

Reply:
no... added examples it it working fine... removing it.